charts/coco-supported/hello-openshift/insecure-policy.rego

```diff
@@ -0,0 +1,26 @@
+package agent_policy
+
+import future.keywords.in
+import input
+
+default CopyFileRequest := false
+default CreateContainerRequest := false
+default CreateSandboxRequest := true
+default DestroySandboxRequest := true
+default ExecProcessRequest := true  # Enabled to allow exec API
+default GetOOMEventRequest := true
+default GuestDetailsRequest := true
+default OnlineCPUMemRequest := true
+default PullImageRequest := true
+default ReadStreamRequest := true   # Enabled to allow log API
+default RemoveContainerRequest := true
+default RemoveStaleVirtiofsShareMountsRequest := true
+default SignalProcessRequest := true
+default StartContainerRequest := true
+default StatsContainerRequest := true
+default TtyWinResizeRequest := true
+default UpdateEphemeralMountsRequest := true
+default UpdateInterfaceRequest := true
+default UpdateRoutesRequest := true
+default WaitProcessRequest := true
+default WriteStreamRequest := false
```

Comment:
use the following policy for insecure - https://github.com/openshift/sandboxed-containers-operator/blob/devel/config/peerpods/podvm/agent-policy/allow-all.rego
The above is incorrect as CopyFileRequest and CreateContainerRequest are needed.